app/views/users/new.html.erb

```diff
@@ -0,0 +1,56 @@
+<!--
+<form  method="POST" action="/users" accept-charset="UTF-8">
+  <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>">
+  <label>Name:</label>
+  <input name="user[name]" type="text"/>
+  <br/>
+  <label>Email:</label>
+  <input name="user[email]" type="email"/>
+  <br/>
+  <label>Password:</label>
+  <input name="user[password]" type="password"/>
+  <br/>
+  <input type="submit" value="Submit"/>
+</form> -->
+<!--
+<%= form_tag '/users', method: :post do %>
```

Comment:
same as line 2 use rails helpers URL.

Reply:
We cannot user Rails helpers on the first two forms because the project forbids us forbidden to use them.